### PR TITLE
Support for typescript

### PR DESCRIPTION
**Pull request by btx0**

### CODE REVIEW

File: browser/javascript.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  env: {
+    browser: true,
+    node: false,
+  },
+  extends: [
+    'airbnb/base',
+    ...[
+      '../lib/rules/jsdoc',
+    ].map(require.resolve),
+  ],
```
Comment (sth-fish) [inline browser/javascript.js]: [
  'airbnb/base',
  '../lib/rules/jsdoc'
].map(require.resolve)

same result but without unnecessary expanding or array

File: browser/typescript.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+/*
+ * Module
+ */
+
+const config = {
+  env: {
+    browser: true,
+    node: false,
+  },
+  extends: [
```
Comment (sth-fish) [inline browser/typescript.js]: [
tslint-config-airbnb',
'../lib/rules/jsdoc'
].map(require.resolve)

File: browser/javascript.js (hunk 1 above)
Comment (btx0) [reply browser/javascript.js]: no, does not work

File: browser/typescript.js (hunk 2 above)
Comment (btx0) [reply browser/typescript.js]: no, does not work

**Review by sth-fish — APPROVED** [review]

**Pull request merged**